build.sbt

```diff
@@ -628,6 +630,57 @@ lazy val sbtProj = (project in file("sbt"))
   )
   .configure(addSbtCompilerBridge)
 
+lazy val sbtBig = (project in file(".big"))
```

Comment:
Is it possible to update the launcher in this PR to use `sbt-big` by default? I think it's important that sbt-big becomes the default since the speedup is most significant for first-time users who don't know anything about sbt. We could then add an option to the launcher so that `sbt -small` (name up for debate) would use the non-big artifact in case of regressions from sbt-big.

Reply:
I think eventually we should rename sbt-big to "sbt".

Reply:
The next step would be releasing some experimental version and see if that breaks community build etc.